docs/data/charts/composition/composition.md

```diff
@@ -71,7 +72,7 @@ It can be used instead of them when there is no need to customize anything outsi
 ### Graphical components
 
 These are any component that render the graphical elements of the chart.
-They are the children of the **Structural components** shown above.
+They are the children of the **SrtsDataProvider, and thus a customized legend. Before, in v7, it was possible to use ChatsLegend inside ChartContainer with together with multiple Line/BarPlots, unfortunately this page as no indication/help/example for that scenario. All graphs rendered here hatructural components** shown above.
```

Comment:
This seems like it was pasted by mistake 😆

Reply:
🙈